src/udev/udev-rules.c

```diff
@@ -1152,13 +1159,15 @@ static void rule_resolve_goto(UdevRuleFile *rule_file) {
                 if (!line->goto_line) {
                         log_error("%s:%u: GOTO=\"%s\" has no matching label, ignoring",
                                   rule_file->filename, line->line_number, line->goto_label);
+                        rule_file_mark_issue(rule_file, LOG_ERR);
 
                         SET_FLAG(line->type, LINE_HAS_GOTO, false);
                         line->goto_label = NULL;
 
                         if ((line->type & ~LINE_HAS_LABEL) == 0) {
                                 log_notice("%s:%u: The line takes no effect any more, dropping",
```

Comment:
This may be converted to `log_warning()`.

Reply:
I think `notice` is enough. I don't think we want to log at warning level for something that is not really an issue.

Reply:
IIUC, the log is shown when some tokens in a non-empty line are dropped and now the line becomes meaningless.
That should be not intentional in most cases, and the line should have some typo or so.
So, `udevadm verify` should detect such lines. Of course, the command reuses the parser, hence it logs about that, but succeeds unfortunately.

I do not think we should keep the log level.

Reply:
Since there is a `log_error()` for the same line above, I don't yet see how upgrading that `log_notice()` to `log_warning()` could be beneficial to the verifier: this wouldn't affect the exit status, and the message is logged anyway.

Reply:
> Since there is a `log_error()` for the same line above, I don't yet see how upgrading that `log_notice()` to `log_warning()` could be beneficial to the verifier: this wouldn't affect the exit status, and the message is logged anyway.

Indeed. I overlooked the first error log.